_topics/en/payment-secret.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Payment secret
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Lightning Network
+  - Privacy Enhancements
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+primary_sources:
+    - title: BOLT4
+    - title: BOLT11
+
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
+  - title: "C-Lightning #3259 adds payment secrets to prevent multipath probing"
+    url: /en/newsletters/2019/12/04/#c-lightning-3259
+
+  - title: 'LND #3788 adds support for "payment addresses" (payment secrets)'
+    url: /en/newsletters/2019/12/11/#lnd-3788
+
+  - title: "Rust-Lightning #893 requires payment secrets to prevent multipath probing"
+    url: /en/newsletters/2021/05/05/#rust-lightning-893
```

Comment:
I noticed that most of the topics seem to give all the mentions ordered by date, but this one seems to be out of order.

Reply:
Fixing, but FYI ordering in the YAML has no effect[1] as the rendering logic date sorts before display.  The actual order I use for these entries is usually "whatever order I saw them in".  :-)

[1] Technically it has an effect when we have two entries with the same date.

Reply:
Ah, very convenient. Thanks, good to know.